source/MaterialXRuntime/RtValue.h

```diff
@@ -145,17 +144,6 @@ class RtValue
         return *_reinterpret_cast<RtIdentifier*>(&_data);
     }
 
-    /// Return const pointer.
-    void* const& asPtr() const
```

Comment:
Remove support for setting pointers as value, as this has never been used. We can add it back later if ever needed.